<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>1130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="chartorgraph">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>17</y>
     <width>591</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>521</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="test_graph"/>
    </widget>
    <widget class="QPushButton" name="chart_see">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>20</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>표로 보기</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTableWidget" name="old_chart">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>531</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="graph_see">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>그래프로 보기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_btn">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>추가</string>
     </property>
    </widget>
    <widget class="QPushButton" name="change_btn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>수정</string>
     </property>
    </widget>
    <widget class="QPushButton" name="del_btn">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>삭제</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>노년부양비</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>유소년부양비</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>총부양비</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="add_edit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>해당 지역 추가 </string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QComboBox" name="last_year">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>177</y>
     <width>76</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2001</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2002</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2003</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2004</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2005</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2006</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2007</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2008</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2009</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2010</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2011</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2012</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2013</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2014</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2015</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2016</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2017</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2018</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2019</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2020</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2021</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>177</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>~</string>
   </property>
  </widget>
  <widget class="QComboBox" name="first_year">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>177</y>
     <width>76</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2001</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2002</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2003</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2004</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2005</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2006</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2007</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2008</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2009</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2010</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2011</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2012</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2013</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2014</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2015</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2016</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2017</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2018</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2019</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2020</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2021</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="rate_test">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>127</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>전국</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>서울특별시</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>부산광역시</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>대구광역시</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>인천광역시</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>광주광역시</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>대전광역시</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>울산광역시</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>세종특별자치시</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>경기도</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>강원도</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>충청북도</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>충청남도</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>전라북도</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>전라남도</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>경상북도</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>경상남도</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>제주특별자치도</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="search_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>227</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>국가별 인구성장률 비교</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="country1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>380</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
background-color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>선택 국가 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="country2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>440</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
background-color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stats">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>560</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>통계</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="graph_stats">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>350</y>
     <width>621</width>
     <height>241</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>x축=연도, y축=인구성장률</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>581</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="world_graph_verticalLayout"/>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>통계</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="stats_tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>591</width>
       <height>111</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_worldChange">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>수정하기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_worldDelete">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>삭제하기</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>420</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>선택 국가 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_yy1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>490</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_yy2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>520</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStackedWidget" name="country_stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>231</width>
     <height>151</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_5">
    <widget class="QLineEdit" name="src_box1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_src1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>검색</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>국가 1 검색</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QListWidget" name="list_country1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>141</width>
       <height>91</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>국가 2 검색</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>검색단어 관련 국가 리스트 나열</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="src_box2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="list_country2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>141</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_src2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>검색</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>검색단어 관련 국가 리스트 나열</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>560</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>초기화</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_graph">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>560</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>그래프</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="modify_city">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>690</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>도시입력</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>650</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>추가</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_button">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>730</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>780</y>
     <width>761</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="graph_verticalLayout"/>
  </widget>
  <widget class="QTableWidget" name="birth_Widget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>660</y>
     <width>761</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="city_comboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>610</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>도시선택</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="modify_birth">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>690</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>합계출산율</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modify_button">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>690</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>수정</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="modify_year">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>690</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>0000년</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="add_city">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>650</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>도시입력</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="del_city">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>730</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>도시입력</string>
   </property>
  </widget>
  <widget class="QLabel" name="head_label">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>600</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;대한민국 2000~2021 합계출산율&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="year1_comboBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>610</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="year2_comboBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>610</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searching_btn">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>610</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_worldAdd">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>국가추가</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_worldAdd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>550</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
